indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -70,7 +70,15 @@ export default function EditableList({management}) {
   const codePresent = Object.values(contribList).some(c => c.code);
   return (
     <EditableListDisplay
-      initialContribList={contribList}
+      initialContribList={
+        (contribList.map(c => {
+          if (c.editable && c.editable.state === 'accepted' && c.editable.submitterAcceptance) {
+            c.editable.state = 'accepted_submitter';
+          }
+          return c;
+        }),
+        contribList)
+      }
```

Comment:
```suggestion
      initialContribList={contribList.map(c =>
        c.editable && c.editable.submitterAcceptance
          ? {...c, editable: {...c.editable, state: 'accepted_submitter'}}
          : c
      )}
```

a bit simpler and it should work just fine